src/components/AffiliationsManager/AffiliationsManager.css

```diff
@@ -0,0 +1,10 @@
+@import "@folio/stripes-components/lib/variables";
```

Comment:
What is the reason to import this style package?

Reply:
Oh, it's really redundant here, thanks! I have already removed that